tasks/windows.task/second-stage.ps1.erb

```diff
@@ -40,5 +41,5 @@ New-Item -ItemType Directory -Force -Path "$($osDrive)\Windows\Setup\Scripts"
 (New-Object System.Net.WebClient).DownloadFile('<%= file_url('SetupComplete.cmd') %>',"$($osDrive)\Windows\Setup\Scripts\SetupComplete.cmd")
 
 write-host "restarting the system to complete installation"
-restart-computer
-start-sleep 30                  # wait for the reboot to complete before exiting
+# wait for the reboot to complete before exiting
+restart-computer -Wait -For PowerShell -Timeout 30
```

Comment:
In general, whenever we've had a timeout like this, we've found that eventually someone will need to override it because stuff is just slow for them. Is that feasible here?